### ISSUE TITLE
Android 3.5.2 crashes when opening a note that uses Secure Spreadsheets

### ISSUE THREAD

**Issue by chaserene** [standardnotes/mobile#308]
Android 9
3.5.2, downloaded from GitHub
[Secure Spreadsheets Github repo](https://github.com/standardnotes/secure-spreadsheets)

it just crashes and quits with the usual Android error message.

**Comment by radko93** [comment]
Are you able to share what device do you use?

**Comment by chaserene** [comment]
if you provide a private way to do that, sure. if it fits you, I can send it to the Protonmail support address.

**Comment by radko93** [comment]
@chaserene sure, please send it there with a link to this thread.

**Comment by TXS380** [comment]
Having the same issues on my tablet(Samsung S5e)  and on my cell phone(Nokia 8).

**Comment by JaspalSuri** [comment]
We weren't able to reproduce this issue on our end yet, but we've taken note of it and will look for any related logs.

**Comment by chaserene** [comment]
@JaspalSuri are you suggesting that multiple other devices you've tested don't exhibit this error?

how can I extract logs for you from the Android app?

**Comment by TXS380** [comment]
Don't know if that helps you but if I create a new note using Secure Spreadsheets on my mobile devices I can work with it until I save it. Once saved I cannot reopen it again without the app crashing. Steps to reproduce:

1. New note
2. Choose editor 'Secure Spreadsheets'
3. Change title
4. fill some cells with random numbers/characters
5. close note
6. click/tap on note in note overview
-> App crashes

**Comment by TXS380** [comment]
Ah forgot:
I can open all notes on my desktop using the appimage for linux.
The standard notes app version is 3.5.3. I could open all these notes before the recent updates.

**Comment by JaspalSuri** [comment]
Hi everyone, I wanted to let you know that we're still looking into this and the other issues that are being reported.

@chaserene thank you for emailing us your device's details. With Bugsnag implemented into the mobile apps (currently no opt-out, since it's technically difficult to remove), we receive anonymized information from users and might be able to view what caused the app to crash. We can't see your notes, IP address, or anything like that. So, in other words, we have the logs and I've pinged our mobile apps dev again to look into it. More details towards the bottom of this [post](https://github.com/standardnotes/forum/issues/1129). 

@TXS380 we've tried that and a similar method but were still unable to reproduce the crash. ☹️ 

I'll be sure to stay in touch with new updates as I come across them.

**Comment by Kes0** [comment]
Hi,
it's same with recent android update (3.5.15 (3000261)).
Created note with secure spreadsheet, closed, trying to open again, app crashed.

No problem to open app on desktop (linux 3.5.11 and 3.5.14).
Also, when I open sheet on desktop, and change editor to plain text, save it, go back to Android App it can't open it and crashes.

**Comment by radko93** [comment]
@Kes0 do you also have Android 9?

**Comment by Kes0** [comment]
@radko93 yes, I have Android 9, Security patch lvl May 1, 2019, Kernel 3.18.120.
It's LG G6 phone

**Comment by Ceaus** [comment]
Can confirm with a very simple spreadsheet. Android crashes all the time.
Android: 3.6.2
AppImage: 3.6.1
